redash/models.py

```diff
@@ -1089,6 +1119,16 @@ def __repr__(self):
         return '<Query %s: "%s">' % (self.id, self.name or 'untitled')
 
 
+class QueryResultSet(db.Model):
+    query_id = Column(db.Integer, db.ForeignKey("queries.id"),
+                      primary_key=True)
+    query_rel = db.relationship(Query)
```

Comment:
I think this should just be `query`, no need for the extra suffix.

Reply:
The suffix is to avoid collision with `db.Model.query`.